change/@azure-msal-angular-a7f4662d-701e-4ffc-bc3d-63067281ab2c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I would mark this as minor given new functionality.

Reply:
Done.